code/modules/surgery/surgery.dm

```diff
@@ -86,10 +86,14 @@
 
 	var/datum/surgery_step/S = get_surgery_step()
 	if(S)
+		var/obj/item/tool = user.get_active_held_item()
 		if(S.try_op(user, target, user.zone_selected, user.get_active_held_item(), src, try_to_fail))
```

Comment:
if(S.try_op(user, target, user.zone_selected, tool, src, try_to_fail))

No need to call the proc twice.